utils/githooks/README.md

```diff
@@ -0,0 +1,19 @@
+# Git hooks
+
+This directory contains githooks that can be employed locally to
+automate things like copyright updates on modified files
+
+To use the pre-commit hook here, do the following locally
+
+```bash
```

Comment:
(lint) Quote this to prevent word splitting. [SC2046]
(lint) Use $(..) instead of legacy `..`. [SC2006]
(lint) Quote this to prevent word splitting. [SC2046]
(lint) Use $(..) instead of legacy `..`. [SC2006]